dateutil/test/test_tz.py

```diff
@@ -14,10 +14,12 @@
 import sys
 import base64
 import copy
+import gc
 
 from functools import partial
 
 IS_WIN = sys.platform.startswith('win')
+NOT_PY3 = not sys.version_info[0] == 3
```

Comment:
Normally you would not have to define this. There's already a `six.PY3` and `six.PY2` that will give you that.

That said, I may have been wrong about weak references being a Python 3 thing. [Seems like they are available in Python 2.7](https://docs.python.org/2.7/library/weakref.html), so we should drop this skip test.